### ISSUE TITLE
Add DemoCamera

### ISSUE THREAD

**Issue by ffvoigt** [mesoSPIM/mesoSPIM-control#16]
This will involve:
* create a `mesoSPIM_Camera` facade class that contains all the methods of the current `mesoSPIM_HamamatsuCamera`
* inherit it in a `mesoSPIM_DemoCamera`  class
* generate some noise demo data to be sent to the `mesoSPIM_CameraWindow`, for example some noisy stripes via:
```Python
def create_demo_image(n, size):
  n = (n % 64) - 32 # the range -32 to 32 produces decent images

  size = 1024 # or 2048
  line = np.linspace(0,6*np.pi,size)
  line = 4*np.sin(line)+4

  data = np.array([np.roll(line, n*i) for i in range(0, size)])
  data = data + np.random.normal(size=(1024, 1024))
```
* one could wire this up to the NI WaveFormGenerator or a Demo variant of it - currently the camera is triggered by a wired digital signal in hardware
*  but as a test, one could simply use the `sig_add_images_to_image_series` signal by the `mesoSPIM_Core` to trigger the generation of images in a simulated stack and `sig_get_live_image` for live mode

**Comment by ffvoigt** [comment]
This feature is now implemented via the full demo mode in the development branch.

**Comment by ffvoigt** [comment]
Released with [0.1.3](https://github.com/mesoSPIM/mesoSPIM-control/releases/tag/0.1.3)
